js/components/utils.js

```diff
@@ -529,6 +529,21 @@ var Utils = {
     return (src && (typeof src === 'string' || src instanceof String))
   },
 
+  /**
+   * Returns sanitized config data
+   *
+   * @function sanitizeConfigData
+   * @param {Object} data to be sanitized
+   * @returns {Object} data if data is valid, empty object if not
+   */
+  sanitizeConfigData: function(data) {
```

Comment:
The name for this is a generic kind of name and hints at it being useful for any data. Do we not accept arrays at all in the skin params config? If not I would rename it.